ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -15,12 +15,14 @@ export class AppVersionsTableContainer extends React.Component {
     const renderedRows = [];
     for (let i = 0; i < versions.length; i++) {
       const tempRow = [];
-
       tempRow[0] = versions[i][0];
       tempRow[1] = versions[i][1];
       tempRow[2] = versions[i][2];
-      tempRow[3] = versions[i][3];
-
+      if (versions[i][3].isNullOrUndefined || versions[i][3] === '') {
+        tempRow[3] = 'Never Launched';
+      } else {
+        tempRow[3] = new Date(versions[i][3]).toString();
```

Comment:
`Thu Mar 14 2019 16:06:30 GMT+0000 (Greenwich Mean Time)`

Is this the decided upon date format? It is very long.

The date format in the flats look like this:

`2019-04-14 16:06:30`

Which I think is preferable. The end-user should just see the date/time as it is in their timezone and not need to do any calculations to convert from GMT to local time.

Thoughts @davidffrench @damienomurchu ?

Reply:
I would rather `2019-04-14 16:06:30`. We are saving the date without timezone in the db. So it should always display UTC.

Reply:
Is there value in displaying the timezone in the date, just so there can be no doubt by the user as to the time? My thinking would be as this is a security service, action may need to be taken in a timely manner, and by displaying the users local timezone in the date the correctness of the date can be ascertained quite quickly